tests/unit/defrag.cpp

```diff
@@ -88,12 +88,49 @@ int iterate_callback(struct eblob_disk_control *dc,
 	return 0;
 }
 
-void check_view_usage(eblob_wrapper &wrapper, size_t sorted_blob_number, size_t unsorted_blob_number) {
+
+struct bases_stats {
+	size_t number_blobs = 0;
+	size_t size_live_data = 0;
+	size_t size_removed_data = 0;
+	size_t number_live_records = 0;
+	size_t number_removed_records = 0;
+};
+
+
+bases_stats calculate_bases_stats_by_bases(eblob_base_ctl **bctls, size_t bctl_cnt) {
+	bases_stats stats;
+	stats.number_blobs = bctl_cnt;
+	for (size_t index = 0; index != bctl_cnt; ++index) {
+		eblob_base_ctl *bctl = bctls[index];
+		size_t number_records = eblob_stat_get(bctl->stat, EBLOB_LST_RECORDS_TOTAL);
+		size_t removed_records = eblob_stat_get(bctl->stat, EBLOB_LST_RECORDS_REMOVED);
+		stats.number_live_records += number_records - removed_records;
+		stats.number_removed_records += removed_records;
+		size_t datasize = eblob_stat_get(bctl->stat, EBLOB_LST_BASE_SIZE) -
```

Comment:
Maybe put a comment why do we need to substract here?

Reply:
I removed subtraction.